app/assets/stylesheets/application.scss

```diff
@@ -2,6 +2,7 @@
 @import 'uswds_overrides/all';
 @import 'c3.min';
 @import 'source/accessible-mega-menu';
+@import 'selectize.bootstrap3';
```

Comment:
Did this require you to define some bootstrap compatible classes in the stylesheet?